build.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-#Update the apt package manager
-sudo apt update || { echo "Updating apt failed"; exit 1; }
```

Comment:
Do we think we might ever need this in the future?

Reply:
Hmm, hard to say. If the hosting VM ever needs this command to be ran, then someone can manually do it